README.md

```diff
@@ -63,5 +63,30 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 
 Run the following
 ```bash
-$ poetry run pytest
+poetry run pytest
+```
+
+## Running in Docker
+
+Build the Docker images by doing:
+```bash
+docker build --target development --tag todo-app:dev .
+docker build --target production --tag todo-app:prod .
+```
+
+You can run the production image by doing
+```bash
+docker run --env-file .env -p 8000:80 todo-app:prod
+```
+
+And you can run the development image by doing
+```bash
+docker run --env-file .env -p 5000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
```

Comment:
Interestingly, I had issues running this in Powershell with the double-quotes. It worked in GitBash and I can't see why it's different from commands that have previously worked, so probably an issue on my machine, but interesting!